ironfish-cli/src/commands/evm/unshield.ts

```diff
@@ -0,0 +1,138 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { isValidAddress } from '@ethereumjs/util'
+import { GoldTokenJson } from '@ironfish/ironfish-contracts'
+import {
+  ContractArtifact,
+  EthSendTransactionRequest,
+  GLOBAL_CONTRACT_ADDRESS,
+} from '@ironfish/sdk'
+import { Flags, ux } from '@oclif/core'
+import { ethers } from 'ethers'
+import { IronfishCommand } from '../../command'
+import { LocalFlags } from '../../flags'
+import { promptCurrency } from '../../utils/currency'
+
+export class UnshieldCommand extends IronfishCommand {
+  static description = `Unshield private asset to public token`
+
+  static flags = {
+    ...LocalFlags,
+    amount: Flags.integer({
+      char: 'a',
+      description: 'The amount to send in the major denomination',
```

Comment:
Are we sending in major or minor?

Reply:
I think it will be minor -- I think the global contract treats the values in minor denomination